Kconfig

```diff
@@ -255,4 +255,11 @@ menu "LittleFS"
         help
             Selects whether littlefs performs runtime assert checks.
 
+    config LITTLEFS_MMAP_PARTITION
+        bool "Memory map LITTLEFS partitions"
+        default "n"
+        help
+            Use esp_partition_mmap to map the partitions to memory, which can provide a significant
+            performance boost in some cases.
```

Comment:
Can you mention the limitations here; you can just use the ESP32 limitations as an example. Basically just tell the user to see if their use-case is compatible with this feature.